public/controlpanel.php

```diff
@@ -279,9 +279,17 @@ function confirmEmailChange(event) {
             echo "<tr>";
             echo "<td class='align-top'>Web API Key</td>";
             echo "<td>";
-            echo "<input class='mb-1' size='60' readonly value='$apiKey'>";
+
+            $copyIcon = Blade::render('<x-fas-copy />');
 
             echo <<<HTML
+                <div role="button" x-init="{}" @click="copyToClipboard('$apiKey')" class="flex items-center gap-x-2 bg-embed max-w-fit rounded pl-2 mb-2">
+                    <p class="font-mono text-neutral-200 light:text-neutral-700">$apiKey</p>
+                    <button class="btn transition-transform lg:active:scale-95" @click="copyToClipboard('$apiKey')" title="Copy web API key to clipboard" aria-label="Copy web API key to clipboard">
+                        $copyIcon
+                    </button>
```

Comment:
The button border blends in with the background and makes it seems unaligned with the text field
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/2e0549c0-e738-442e-93b5-03a198cdf65b)

Reply:
![Screenshot 2024-01-13 at 7 00 41 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/59de3022-7cb6-4a18-804a-a80b7c568302)

I've made an adjustment in latest such that it's no longer ambiguous where the button is. There's now a single button, with the key and icon contained therein.